docs/release_notes/release_notes.md

```diff
@@ -8,111 +8,45 @@ has_children: true
-        * Email: info@crossrealms.com / support@crossrealms.com
-
-* ### Added New Alerts
-    * Active Directory
-        * AD - Password Change Outside Working Hour
-    * Cisco IOS
-        * Cisco IOS - New Connection For User
-        * Cisco IOS - Device Failed Login
-    * Office 365
-        * O365 - Authentication Blocked By Conditional Access Policy
-        * O365 - Daily Login Failure
-        * O365 - Login Failure Due To Multi Factor Authentication
-        * O365 - Login Failure Outside Home Country Due To Multi Factor Authentication
-        * O365 - Login From Unknown User
-        * O365 - Security Compliance Alert
-        * O365 - Successful Login Outside Home Country
-    * Office 365 Active Directory
-        * O365 - Azure Active Directory - GroupMembership Change/Update
-    * Office 365 Emails
-        * Email- Hourly Increase In Emails Over Baseline
-        * Email- Daily Spam Email
-    * Palo Alto
-        * Palo Alto Firewall - Commits
-    * VPN
-        * Authentication - Successful VPN Login Outside Home Country
-
-
-
-* ### Enhancements
-    * Office 365 Dashboard
-        * Added a new alert and dashboard panel called `Azure Active Directory - GroupMembership Change/Update`.
-
-    * Antivirus details have been added to the Lansweeper Dashboard.
-        * Added an `active_antivirus` column to multiple dashboard panels.
-        * Users needs to have version 1.3.0 of the Lansweeper Add-on to collect the Antivirus related data.
-
-    * Updated the following Antivirus releated alerts to reduce the number of false positives using Lansweeper's asset data. The updated alerts will filter the host if it has any other antivirus enabled on it.
-        * Sophos - Endpoint Not Protected by Sophos
-        * Sophos - Sophos Service is not Running
-        * Windows Defender - Windows Defender RealTime Protection Disabled or Failed
-
-    * Enhanced the search query for the `Windows Defender - Windows Defender RealTime Protection Disabled or Failed` alert to reduce the number of false positives.
-        * Changed the default cron job from every 15 minutes to every hour.
-
-    * Bruteforce alerts
-        * Added the dest field to the results of the following bruteforce related alerts for a more thorough forensic investigation:
-            * Authentication - Bruteforce Attempt for a User
-            * Authentication - Bruteforce Attempt from a Source
-            * Authentication - Excessive Failed VPN Logins for a User
-            * Authentication - Excessive Failed VPN Logins from a Source
-
-    * Better support for DNS logs
-        * Added props and field extractions for Windows DNS Logs (MSAD:NT6:DNS, isc:bind:query, & isc:bind:queryerror sourcetypes).
+## Version 2.1.0 (May 2022)
 
 
+* ### Ransomware alert improvement
+    * Ignored below file paths in the `Ransomware - Calculate UpperBound for Spike in File Writes` and `Ransomware - Spike in File Writes` alerts to reduce the false positive.
```

Comment:
Added filters for paths to reduce false positives for both report `Ransomware - Calculate UpperBound for Spike in File Writes` and alert `Ransomware - Spike in File Writes`.